urllib3/response.py

```diff
@@ -93,14 +93,19 @@ class is also compatible with the Python standard library's :mod:`io`
     :param retries:
         The retries contains the last :class:`~urllib3.util.retry.Retry` that
         was used during the request.
+
+    :param strict_content_length:
+        Enforce content length checking. Body returned by server must match
+        value of Content-Length header, if present. Otherwise, raise error.
     """
 
     CONTENT_DECODERS = ['gzip', 'deflate']
     REDIRECT_STATUSES = [301, 302, 303, 307, 308]
 
     def __init__(self, body='', headers=None, status=0, version=0, reason=None,
                  strict=0, preload_content=True, decode_content=True,
-                 original_response=None, pool=None, connection=None, retries=None):
+                 original_response=None, pool=None, connection=None,
+                 strict_content_length=False, retries=None, request_method=None):
```

Comment:
Should request_method be replaced with **response_kw so we're not defining minimal contact params?